src/main/java/killrvideo/service/VideoCatalogService.java

```diff
@@ -240,27 +247,29 @@ public void submitYouTubeVideo(SubmitYouTubeVideoRequest request, StreamObserver
         batchStatement.add(insertLatestVideo);
         batchStatement.setDefaultTimestamp(now.getTime());
 
-        FutureUtils.buildCompletableFuture(session.executeAsync(batchStatement))
+        CompletableFuture<ResultSet> insertUserFuture = FutureUtils.buildCompletableFuture(session.executeAsync(batchStatement))
                 .handle((rs, ex) -> {
                     if (rs != null) {
                         /**
                          * See class {@link VideoAddedHandlers} for the impl
                          */
                         final YouTubeVideoAdded.Builder youTubeVideoAdded = YouTubeVideoAdded.newBuilder()
                                 .setAddedDate(TypeConverter.dateToTimestamp(now))
-                                .setDescription(request.getDescription())
+                                .setDescription(description)
                                 .setLocation(location)
-                                .setName(request.getName())
+                                .setName(name)
                                 .setPreviewImageLocation(previewImageLocation)
                                 .setTimestamp(TypeConverter.dateToTimestamp(now))
                                 .setUserId(request.getUserId())
                                 .setVideoId(request.getVideoId());
 
-                        youTubeVideoAdded.addAllTags(Sets.newHashSet(request.getTagsList()));
+                        youTubeVideoAdded.addAllTags(Sets.newHashSet(tagsList));
 
                         /**
-                         * eventbus.post() below is located in the VideoAddedhandlers class within
-                         * the handle() method.  The youTubeVideoAdded type triggers the handler.
+                         * eventbus.post() for youTubeVideoAdded below is located both in the
+                         * VideoAddedhandlers and SuggestedVideos Service classes within the handle() method.
+                         * The YouTubeVideoAdded type triggers the handler.  The call in SuggestedVideos is
```

Comment:
I'm a purist I guess but having comments referencing who the subscriber of an event is seems to violate some principle of separation.

Reply:
I am doing this for those that might not be familiar with what is going on there.  I'm trying to save folks from having to look all over the place figuring out what is going on.